operations/system_management_health/Remove_Kiali.md

```diff
@@ -0,0 +1,22 @@
+# Remove Kiali
```

Comment:
should we update customization instead of manually remove it? if they don't want it we should make sure once this is done, it won't come back during upgrade